### PR TITLE
Adds cookie consent functionality 

### PR DESCRIPTION
**Pull request by LucasKnudsen**
pt story: https://www.pivotaltracker.com/story/show/178593650

## User Story 

```
As a User
I want to be able to decided whether to send cookies
I want to be able to accept or decline cookies
```

## Suggested changes

- Adds cookie acceptance form on page 

##  Dependencies

-  react-cookie-consent

## What I learned 

- How to add cookie consent to a webpage

## Screenshot

https://user-images.githubusercontent.com/63557569/122636703-e6522580-d0ea-11eb-8151-d68565aae324.mp4

### CODE REVIEW

File: src/components/Header.jsx [hunk 1]
```diff
@@ -23,8 +25,8 @@ const LandingHeader = ({ landing }) => {
         <div className='menu-wrapper'>
           <CustomButton
             dataCy='phone-question'
-            onClick={ahoy.track(`phone_button`)}>
-            <a href='tel:+460317674100'>031-767 41 00</a>
+            onClick={consent && ahoy.track(`phone_button`)}>
+            <a href='tel:+46311234567'>031-123 45 67</a>
```
Comment (1CIM) [inline src/components/Header.jsx]: think it should be 0317674100? No?

File: src/components/Footer.jsx [hunk 2]
```diff
@@ -46,8 +48,8 @@ const Footer = () => {
           <h3 data-cy='footer-contact-header'>{t('footer.subHeader2')}</h3>
           <CustomButton
             dataCy='phone-number'
-            onClick={ahoy.track(`phone_button`)}>
-            <a href='tel:+46317674100'>031-767 41 00</a>
+            onClick={consent && ahoy.track(`phone_button`)}>
+            <a href='tel:+4631192342'>031-19 23 42</a>
```
Comment (1CIM) [inline src/components/Footer.jsx]: and here?

File: src/components/Header.jsx (hunk 1 above)
Comment (DavDan1) [reply src/components/Header.jsx]: just change here @lukeperrin10 maybe ?

Comment (lukeperrin10) [reply src/components/Header.jsx]: fixed!

File: src/components/Footer.jsx (hunk 2 above)
Comment (lukeperrin10) [reply src/components/Footer.jsx]: fixed!

**Pull request merged**